gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnAppLauncher.java

```diff
@@ -608,21 +608,22 @@ private Resource prepareContainerResource(GetNewApplicationResponse newApplicati
     Path appMasterWorkDir = new Path(appWorkDir, GobblinYarnConfigurationKeys.APP_MASTER_WORK_DIR_NAME);
 
     Map<String, LocalResource> appMasterResources = Maps.newHashMap();
+    FileSystem localFs = FileSystem.getLocal(this.yarnConfiguration);
```

Comment:
Maybe it is safer to use FileSystem.getLocal(new Configuration()) since we are dependent of fs.defaultFS not being configured in yarnConfiguration.